### ISSUE TITLE
ArchieML 1.1

### ISSUE THREAD

**Issue by thatrobert** [newsdev/archieml.org#47]
Are there plans to further refine the ArchieML specification?  An obvious upgrade would be to support all JSON to be ArchieML compatible when converting back and forth using a tool like Jughead.

The obvious issues with converting ArchieML to JSON are:
- Allowing spaces within keys
- Support for numbers, bools, and NULLs

One straightforward solution would be to add quotes in those situations to clarify intent. So this block of JSON:
{
   "property name": NULL,
   "cost": 34.01,
   "Not a null": "null",
   "IsShipped": true,
   "String list": [
      "hi",
      "bye"
   ]
}

Would become the following ArchieML:

"property name": NULL
cost: 34.01
"Not a null": "null"
IsShipped: true
["String list"]
* hi
* bye

In other words, only use quotes when you have to.  And if numbers, bools, or NULLs should be treated as strings, put them in quotes.
